byterun/caml/mlvalues.h

```diff
@@ -108,19 +108,19 @@ bits  63        (64-P) (63-P)        10 9     8 7   0
 #define PROFINFO_MASK ((1ull << PROFINFO_WIDTH) - 1ull)
 
 #define Tag_hd(hd) ((tag_t) ((hd) & 0xFF))
-#ifndef WITH_SPACETIME
-#define Wosize_hd(hd) ((mlsize_t) ((hd) >> 10))
-#else
+#ifdef WITH_SPACETIME
 #define Hd_no_profinfo(hd) ((hd) & ~(PROFINFO_MASK << PROFINFO_SHIFT))
 #define Wosize_hd(hd) ((mlsize_t) ((Hd_no_profinfo(hd)) >> 10))
-#endif
+#else
+#define Wosize_hd(hd) ((mlsize_t) ((hd) >> 10))
+#endif /* SPACETIME */
 #ifdef ARCH_SIXTYFOUR
 /* [Profinfo_hd] is used when the compiler is not configured for Spacetime
    (e.g. when decoding profiles). */
 #define Profinfo_hd(hd) (((mlsize_t) ((hd) >> PROFINFO_SHIFT)) & PROFINFO_MASK)
 #else
 #define Profinfo_hd(hd) ((hd) & 0)
-#endif
+#endif /* ARCH_SIXTYFOUR */
 
```

Comment:
There is currently no bug, since PROFINFO_WIDTH can only take value 26, and the hardcoded value is 26. So, strictly speaking, there is no need for this bug fix in 4.04. For me, very small bug fixes and code cleanup like these ones can take place during the release process, as this process is mostly documentating and bug hunting.